docs/overview/excel.md

```diff
@@ -63,32 +63,17 @@ Our [tutorials](../tutorials/excel-tutorial.md) provide a guided and structured
 
 ## Share scripts
 
+Office Scripts can be shared with other users of an Excel workbook. When you share a script in a shared workbook, everyone with access to the workbook can also view and run your script. For more details about sharing and unsharing scripts, see [Sharing Office Scripts in Excel for the Web](https://support.microsoft.com/office/226eddbc-3a44-4540-acfe-fccda3d1122b).
+
 :::image type="content" source="../images/script-sharing.png" alt-text="The script Details page showing the 'Share with others in this workbook' option.":::
 
-Office Scripts can be shared with other users of an Excel workbook. When you share a script in a shared workbook, everyone with access to the workbook can also view and run your script.
+Add buttons that run scripts to help your colleagues discover your helpful solutions. Learn more about script buttons in [Run Office Scripts with buttons](../develop/script-buttons.md).
```

Comment:
Another word to avoid redundancy? 
```suggestion
Add buttons that run scripts to help your colleagues discover your valuable solutions. Learn more about script buttons in [Run Office Scripts with buttons](../develop/script-buttons.md).
```